src/elements/alt-style/AltStyleElementForm.tsx

```diff
@@ -45,10 +43,13 @@ const ChildNumber = styled("div")`
   display: flex;
   justify-content: center;
   align-items: center;
-  height: 24px;
+  height: 32px;
   padding: 2px;
-  width: 24px;
+  width: 32px;
   border: 1px solid ${neutral[60]};
+  position: absolute;
+  top: 0;
+  left: -32px;
```

Comment:
Should this number be inside `LeftRepeaterActions` to avoid repetition (and not having to maintain the negative positioning in more than one place?

Reply:
I had it in there to start with, but wanted the Child Number to always be visible so moved it out (if it's in actions we can only see it on hover).

Reply:
makes sense, thanks for clarifying. perhaps worth moving the -32 to a `const`?